README.md

```diff
@@ -179,14 +189,75 @@ applications:
   - conjur
 ```
 
+In PCF version 2.0+, when the service broker creates the Host identity for your application
+in Conjur, it will automatically add it to a Conjur Layer representing the `Organization`
+and `Space` the application is deployed into on PCF. This allows `Spaces` to be privileged
+and any apps deployed into that space to inherit those privileges.
+
+To accomplish this the service broker requires that the Org and Space policy exist prior
+to binding applications with Conjur. The policy to define an organizaiton and space is:
+```yaml
+---
+# Policy for the Organization
+- !policy
+  # Organization GUID from PCF.
+  # This may be obtained by running `cf org --guid {org name}
+  id: cbd7a05a-b304-42a9-8f66-6827ae6f78a1
+  body:
+    # Layer to allow privileging an entire organzation to a resource
+    - !layer
+
+    # Policy for the Space
+    - !policy
+      # Space GUID from PCF.
+      # This may be obtained by running `cf space --guid {space name}
+      id: 8bf39f4a-ebde-437b-9c38-3d234b80631a
+      body:
+        # Layer to allow privileging an entire space to a resource
+        # The service broker will add applications to this layer automatically.
+        - !layer
+
+    # Grant to add the Space layer to the Org Layer
+    - !grant
+      role: !layer
+      member: !layer 8bf39f4a-ebde-437b-9c38-3d234b80631a
+
+# Allow the service broker to manage the Organization policy
+- !permit
+  resource: !policy cbd7a05a-b304-42a9-8f66-6827ae6f78a1
+  privileges: [ create, update, read ]
+  # This should be the service broker identity you created with configuring the service broker
+  roles: !host service-broker
```

Comment:
are these permits actually necessary? the service broker is a member of the group that owns the CONJUR_POLICY branch - I would expect that they inherit ownership of the space / org policy branches too.

Reply:
Ah, I didn't have the pre-requisite policy set up correctly. Yes, when the ownership is set up as documented, these are not necessary. 

I will remove them. Thanks!